javascript/builtins/Error.json

```diff
@@ -393,7 +393,7 @@
                 "version_added": true
               },
               "edge": {
-                "version_added": true
+                "version_added": false
```

Comment:
I don't believe this is accurate.  Edge supports "stack".

Reply:
@concavelenz would you mind opening a new issue or PR about this? It's very likely for this suggestion to get lost otherwise